### ISSUE TITLE
v0.20.0: Post https://tekton-triggers-core-interceptors.tekton-pipelines.svc:80/cel network is unreachable

### ISSUE THREAD

**Issue by r0bj** [tektoncd/triggers#1370]
# Expected Behavior
Triggers work after tekton triggers update to version v0.20.0.

# Actual Behavior
Triggers don't work after tekton triggers update to version v0.20.0.
Event generates error logs message:
```
{"level":"error","ts":"2022-05-25T23:23:06.120Z","logger":"eventlistener","caller":"sink/sink.go:381","msg":"Post \"https://tekton-triggers-core-interceptors.tekton-pipelines.svc:80/cel\": dial tcp 10.220.28.32:80: connect: network is unreachable","eventlistener":"kustomize-build","namespace":"tekton-execution","/triggers-eventid":"16977504-7f18-4734-ab12-7474e94da8a7","eventlistenerUID":"da296abe-d7a2-44dd-ab33-02e9908bd79d","/triggers-eventid":"16977504-7f18-4734-ab12-7474e94da8a7","/trigger":"kustomize-build-cron","stacktrace":"github.com/tektoncd/triggers/pkg/sink.Sink.processTrigger\n\tgithub.com/tektoncd/triggers/pkg/sink/sink.go:381\ngithub.com/tektoncd/triggers/pkg/sink.Sink.HandleEvent.func1\n\tgithub.com/tektoncd/triggers/pkg/sink/sink.go:196"}
```

Request is sent to URL https://tekton-triggers-core-interceptors.tekton-pipelines.svc:80/cel but service `tekton-triggers-core-interceptors.tekton-pipelines.svc` is expecting traffic on port 8443 (https) instead of on port 80 (http) so there is network is unreachable error generated.

- Tekton Pipeline version:
```
Client version: 0.23.1
Pipeline version: v0.35.0
Triggers version: v0.20.0
```

**Comment by khrm** [comment]
Did you upgrade incrementally? From which version did you upgrade?

**Comment by r0bj** [comment]
Yes, I upgraded from version v0.19.1 to v0.20.0 (and many incremental upgrades before that).

**Comment by flo-02-mu** [comment]
I'm running into the same issue. The ClusterInterceptors from the installation guide (e.g. https://storage.googleapis.com/tekton-releases-nightly/triggers/latest/interceptors.yaml) are setup like this:
```yaml
apiVersion: triggers.tekton.dev/v1alpha1
kind: ClusterInterceptor
metadata:
  name: cel
spec:
  clientConfig:
    service:
      name: tekton-triggers-core-interceptors
      namespace: tekton-pipelines
      path: "cel"
```
which does not use https on port 8443
I got it up an running using:
```yaml
apiVersion: triggers.tekton.dev/v1alpha1
kind: ClusterInterceptor
metadata:
  name: cel
spec:
  clientConfig:
    url: https://tekton-triggers-core-interceptors:8443/cel
```
instead.

**Comment by znqerz** [comment]
I met the same issue on v0.20.0 as @flo-02-mu said.

**Comment by dibyom** [comment]
Duplicate of #1374 
/assign @savitaashture

**Comment by savitaashture** [comment]
Hi @r0bj Sorry for the late response

Yes there is a PR https://github.com/tektoncd/triggers/pull/1379 which shows that port should be same in both clusterinterceptor spec as well as service

Please do let me know if you still facing this issue

**Comment by znqerz** [comment]
the issue was fixed after upgraded to triggers version: v0.20.1.

**Comment by savitaashture** [comment]
> the issue was fixed after upgraded to triggers version: v0.20.1.

Thank you @znqerz for trying out with 0.20.1

Shall i close the issue ?

**Comment by savitaashture** [comment]
Closing issue as issue got resolved with `v0.20.1`
